iphone/TitaniumKit/TitaniumKit/Sources/API/TiViewProxy.m

```diff
@@ -768,6 +773,7 @@ - (TiPoint *)convertPointToView:(id)args
   TiPoint *tiPoint = [[TiPoint alloc] autorelease];
   [tiPoint setX:NUMFLOAT(convertDipToDefaultUnit(pointOffsetDips.x + givenPoint.x))];
   [tiPoint setY:NUMFLOAT(convertDipToDefaultUnit(pointOffsetDips.y + givenPoint.y))];
+  [tiPoint add:[arg2 contentOffset]];
```

Comment:
I think we can replace above line with following code and all other changes can be discarded.
```suggestion
   if ([arg2 respondsToSelector:@selector(contentOffset)]) {
    // In scrollview contentOffset should be applied to returned value
    TiPoint *contentOffset = [arg2 contentOffset];
    [tiPoint setX:NUMFLOAT(convertDipToDefaultUnit(pointOffsetDips.x + contentOffset.point.x + givenPoint.x))];
    [tiPoint setY:NUMFLOAT(convertDipToDefaultUnit(pointOffsetDips.y + contentOffset.point.y + givenPoint.y))];
  } else {
    [tiPoint setX:NUMFLOAT(convertDipToDefaultUnit(pointOffsetDips.x + givenPoint.x))];
    [tiPoint setY:NUMFLOAT(convertDipToDefaultUnit(pointOffsetDips.y + givenPoint.y))];
  }
```

Reply:
Do we need to update doc as well?